Remora.Results/Result.cs

```diff
@@ -32,6 +32,17 @@ namespace Remora.Results;
 [PublicAPI]
 public readonly struct Result : IResult
 {
+    /// <summary>
+    /// Gets a successful result.
+    /// </summary>
+    /// <remarks>
+    /// <para>
+    /// This property returns a result whose <see cref="Error"/> property is set to null, therefore <see cref="IsSuccess"/> returns true.
```

Comment:
You can remove this paragraph - these are implementation details and redundant information for this particular property.

Reply:
Removed the paragraph.